tests/flow/test_string_pool.py

```
@@ -2,26 +2,49 @@
+
+    # GRAPH.INFO ObjectPool
+    # 1) "Object Pool"
+    # 2) 1) 1) "Unique Objects in Pool"
+    # 2) (integer) 1
+    # 2) 1) "Average References per Object"
+    # 2) "2"
+
+    objs_in_pool   = int(stats[1][0][1])
+    avg_ref_count = float(stats[1][1][1])
+
+    assert(avg_ref_count, avg)
```

Ruff (0.11.9)

Assert test is a non-empty tuple, which is always `True`

(F631)